core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3922,6 +3922,21 @@ public String toString() {
               + "aggregated, so different applications must set their own ids or leave this value "
               + "unset to use a randomly generated id.")
           .build();
+  public static final PropertyKey USER_SKIP_AUTHORITY_CHECK =
+      new Builder(Name.USER_SKIP_AUTHORITY_CHECK)
+          .setScope(Scope.CLIENT)
+          .setDefaultValue(false)
+          .setIsHidden(true)
```

Comment:
I believe this property is better made hidden, to avoid confusing. This should ONLY be used to work around thorny legacy code that is too hard to update the URL. This should be used sparingly only under our recommendation.